zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZkClientUriDomainMappingHelper.java

```diff
@@ -72,14 +72,15 @@ public ZkClientUriDomainMappingHelper(ZooKeeperServer zks) {
 
     this.rootPath =
         X509AuthenticationConfig.getInstance().getZnodeGroupAclClientUriDomainMappingRootPath();
+    LOG.debug("Client URI domain mapping root path: {}", this.rootPath);
```

Comment:
nit: To make the logs more consistent, please follow the styles of the loggings originally in this class - Have the class name at the beginning of the log message.